lcls-twincat-motion/Library/POUs/Motion/Utils/FB_SetEnables.TcPOU

```diff
@@ -1,16 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
-<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4022.18">
+<TcPlcObject Version="1.1.0.1" ProductVersion="3.1.4024.12">
   <POU Name="FB_SetEnables" Id="{9028a506-3476-4ab7-9f18-be1b77c07efa}" SpecialFunc="None">
     <Declaration><![CDATA[FUNCTION_BLOCK FB_SetEnables
 // Update the all enable booleans based on the booleans that make them up
 VAR_IN_OUT
     stMotionStage: DUT_MotionStage;
 END_VAR]]></Declaration>
     <Implementation>
-      <ST><![CDATA[stMotionStage.bAllForwardEnable := stMotionStage.bLimitForwardEnable AND (stMotionStage.bGantryForwardEnable OR NOT stMotionStage.bGantryAxis);
-stMotionStage.bAllBackwardEnable := stMotionStage.bLimitBackwardEnable AND (stMotionStage.bGantryBackwardEnable OR NOT stMotionStage.bGantryAxis);
+      <ST><![CDATA[stMotionStage.bAllForwardEnable := stMotionStage.bLimitForwardEnable AND (stMotionStage.bGantryForwardEnable OR NOT stMotionStage.bGantryAxis) AND stMotionStage.stEPSForwardEnable.bEPS_OK;
+stMotionStage.bAllBackwardEnable := stMotionStage.bLimitBackwardEnable AND (stMotionStage.bGantryBackwardEnable OR NOT stMotionStage.bGantryAxis) AND stMotionStage.stEPSBackwardEnable.bEPS_OK;
 
-stMotionStage.bAllEnable := stMotionStage.bEnable AND stMotionStage.bHardwareEnable;
+stMotionStage.bAllEnable := stMotionStage.bEnable AND stMotionStage.bHardwareEnable AND stMotionStage.stEPSPowerEnable.bEPS_OK;
```

Comment:
Reviewer's note: this defaults to TRUE if no FB_EPS is used, maintaining backwards compatibility :+1: